package.json

```diff
@@ -181,10 +181,11 @@
 		"publish:check": "npm run build:packages && lerna updated",
 		"publish:dev": "npm run build:packages && lerna publish --npm-tag next",
 		"publish:prod": "npm run build:packages && lerna publish",
-		"test": "npm run lint && npm run test-unit",
+		"test": "npm run lint && npm run test-unit && npm run test-mobile",
 		"pretest-e2e": "concurrently \"./bin/reset-e2e-tests.sh\" \"npm run build\"",
 		"test-e2e": "cross-env JEST_PUPPETEER_CONFIG=test/e2e/puppeteer.config.js wp-scripts test-unit-js --config test/e2e/jest.config.json --runInBand",
 		"test-e2e:watch": "npm run test-e2e -- --watch",
+		"test-mobile": "cd gutenberg-mobile; yarn test:inside-gb; cd ..;",
```

Comment:
Hmm, no I haven't 😬 

I'd be inclined to modify this into a single `yarn --cwd gutenberg-mobile test-mobile` command if you think it can be problematic in its current form.